database_things.py

```diff
@@ -25,29 +25,28 @@ def make_tables():
 
 
 class Address(Base):
-	"""Retaining the unique component of the address in the address table, using additional tables 
-	for city, state et cetera to accommodate later expansion into other markets as need be."""
-	
-	__tablename__ = "Addresses"
-
-	id = Column(Integer, primary_key=True)
-	name = Column(String, nullable=False)
-	name_secondary = Column(String, nullable=True)
-	user_id = Column(Integer, ForeignKey("Users.id"), nullable=False)
-	is_billing = Column(Boolean, nullable=False)
-	city_id = Column(Integer, ForeignKey("Cities.id"), nullable=False)
-	state_id = Column(Integer, ForeignKey("States.id"), nullable=False)
-	zipcode_id = Column(Integer, ForeignKey("Zipcodes.id"), nullable=False)
-	latitude = Column(String, nullable=False)
-	longitude = Column(String, nullable=False)
-
-	username = relationship("User", backref=backref("addresses"))
-	city_name = relationship("City", backref=backref("addresses"))
-	state_abbreviation = relationship("State", backref=backref("addresses"))
-	zipcode = relationship("Zipcode", backref=backref("addresses")) 
-
-	def __repr__(self):
-		return "<Address object %s>" % self.name
+    """Retaining the unique component of the address in the address table, using additional tables
+    for city, state et cetera to accommodate later expansion into other markets as need be."""
+
+    __tablename__ = "Addresses"
+    id = Column(Integer, primary_key=True)
+    name = Column(String, nullable=False)
+    name_secondary = Column(String, nullable=True)
+    user_id = Column(Integer, ForeignKey("Users.id"), nullable=False)
+    type = Column(String, nullable=False)
```

Comment:
Sorry this diff is really confusing. I changed the column "is_billing" for "type" so that way you can have more then just two type of address.